lib/ret_web/templates/page/avatar-meta.html.eex

```diff
@@ -0,0 +1,16 @@
+<meta property="og:type" content="website" />
+<meta property="og:url" content="<%= @avatar |> Ret.Avatar.url %>" />
+<meta property="og:title" content="<%= @avatar.name %> | Hubs by Mozilla" />
+<meta property="og:description" content="<%= @avatar.description %>" />
+<meta property="og:image" content="<%= @avatar.thumbnail_owned_file |> Ret.OwnedFile.uri_for |> URI.to_string %>"/>
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:domain" value="<%= RetWeb.Endpoint.host %>" />
+<meta name="twitter:title" value="<%= @avatar.name %> | Hubs by Mozilla" />
+<meta name="twitter:description" content="<%= @avatar.name %> is an avatar you can use in Hubs." />
```

Comment:
Maybe put the avatar name in quotes? So it reads like `"Foo Bar" is an avatar you can use in Hubs.`